### DISCUSSION TITLE
SoundFont 1.0 support?

### DISCUSSION THREAD

**Discussion by Sembiance** [FluidSynth/fluidsynth#742] [Ideas]
Greetings

I recently came across a SoundFont 1.0 file: https://telparia.com/fileFormatSamples/audio/soundFont1/SYNTHGM.SBK

Any chance of getting SoundFont 1.0 support added to fluidsynth?

Or maybe a better question, does anyone know of a converter that can take a 1.0 sound font and convert it to 2.0?

PS: Fluidsynth is great! Do you have a way to accept donations?

**Comment by derselbst** [comment]
> Any chance of getting SoundFont 1.0 support added to fluidsynth?

No, because there is hardly anything to support here. SoundFont 1.0 was closely bound to the hardware of the AWE32 and as such had many quirks that made it unportable. That's why EMU invented SF2. I've never seen a spec of SF1 either.

> does anyone know of a converter that can take a 1.0 sound font and convert it to 2.0?

* Creative's Vienna Soundfont Studio
* [Awave Studio](https://www.fmjsoft.com/awavestudio.html#main)
* [sf1to2](http://www.ibiblio.org/thammer/HammerSound/localfiles/soundfonts/sf1to2.zip)  (you'll need DosBox... unless you still have a Win9x running around)
* [other tools may work as well, see the wiki](https://github.com/FluidSynth/fluidsynth/wiki/SoundFont#conversion-tools)

The readme of sf1to2 is very informative. Also, don't expect too much from the font you linked. It is only 35 KiB in size. It's unlikely that it contains _any_ sample. I would not be surprised if it only consisted of ROM samples. In this case, you would need an AWE32 to produce any sound... probably even to convert it.

> Fluidsynth is great!

Thanks!

> Do you have a way to accept donations?

Fluidsynth as a whole does not. Individual contributors might still do, but I don't know of any.

**Reply by Sembiance** [reply]
Thank you VERY much for the information and the links to Awave Studio. You were correct, it only had ROM samples inside. Thanks again!
